azure-pipelines-ui-tests.dfe.yml

```diff
@@ -224,3 +225,212 @@ jobs:
+
+  - task: NodeTool@0
+    displayName: Use Node version 18 above
+    timeoutInMinutes: 5
+    inputs:
+      versionSpec: "18.18.2"
+
+  - script: |
+        node -v
+    displayName: 'node version'
+
+  - script: |
+      npm ci
+    displayName: 'Install CI'
+    workingDirectory: $(System.DefaultWorkingDirectory)/tests/playwright-tests/
+
+  - script: |
+      npx playwright install --with-deps
+    displayName: 'Install Playwright browsers'
+    workingDirectory: $(System.DefaultWorkingDirectory)/tests/playwright-tests/
+        # runs the playwright tests
+
+  - task: AzureKeyVault@2
+    displayName: 'Azure Key Vault: s101d01-kv-ees-01'
+    inputs:
+      azureSubscription: $(SPN_NAME)
+      KeyVaultName: s101d01-kv-ees-01
+      SecretsFilter: ees-test-SLACK-BOT-TOKEN,ees-test-SLACK-TEST-REPORT-WEBHOOK-URL
+      RunAsPreJob: true
+
+  - script: |
+      npx playwright test --project "public"
+    displayName: "Run Playwright tests"
+    workingDirectory: $(System.DefaultWorkingDirectory)/tests/playwright-tests/
+    env:
+            CI: 'true'
+            PUBLIC_URL: $(PUBLIC_URL)
+            PUBLIC_USERNAME: $(PUBLIC_AUTH_USER)
+            PUBLIC_PASSWORD: $(PUBLIC_AUTH_PASSWORD)
+            ADMIN_URL: $(ADMIN_URL)
+            ADMIN_EMAIL: $(ADMIN_EMAIL)
+
+    # this publishes the html report as an artifact that can then be downloaded
+  - task: PublishTestResults@2
+    displayName: Publish Test Results
+    inputs:
+      testResultsFiles: tests/playwright-tests/test-results/playwright-results.xml
+      failTaskOnFailedTests: true
+
+  - task: PublishPipelineArtifact@1
+    displayName: Publish Pipeline Artifact
+    condition: succeededOrFailed()
+    inputs:
+      path: tests/playwright-tests/test-results/
+      artifactName: test-results-playwright-tests-public
+
+
+- job: 'AdminPlaywrightUItest'
+  displayName: Admin suite - Playwright UI tests
+  timeoutInMinutes: 160
+  cancelTimeoutInMinutes: 10
+  condition: succeededOrFailed()
+  pool: ees-ubuntu2204-large
+  workspace:
+    clean: all
+  steps:
+  - checkout: self
+    clean: true
+    fetchDepth: 5
+    fetchTags: false
+
+  - task: NodeTool@0
+    displayName: Use Node version 18 above
+    timeoutInMinutes: 5
+    inputs:
+      versionSpec: "18.18.2"
+
+  - script: |
+        node -v
+    displayName: 'node version'
+
+  - script: |
+      npm ci
+    displayName: 'Install CI'
+    workingDirectory: $(System.DefaultWorkingDirectory)/tests/playwright-tests/
+
+  - script: |
+      npx playwright install --with-deps
+    displayName: 'Install Playwright browsers'
+    workingDirectory: $(System.DefaultWorkingDirectory)/tests/playwright-tests/
+        # runs the playwright tests
+
+  - task: AzureKeyVault@2
+    displayName: 'Azure Key Vault: s101d01-kv-ees-01'
+    inputs:
+      azureSubscription: $(SPN_NAME)
+      KeyVaultName: s101d01-kv-ees-01
+      SecretsFilter: ees-test-ADMIN-PASSWORD,ees-test-SLACK-BOT-TOKEN,ees-test-SLACK-TEST-REPORT-WEBHOOK-URL
```

Comment:
Looks like we're not using the `ees-test-SLACK-BOT-TOKEN` or `ees-test-SLACK-TEST-REPORT-WEBHOOK-URL` secrets here, so they can probably go.

Same comment for KeyVault jobs further down in this file that are used for Playwright.